README.md

```diff
@@ -20,13 +20,42 @@ limitations under the License.
 
 - [Text Generation Inference on Habana Gaudi](#text-generation-inference-on-habana-gaudi)
   - [Table of contents](#table-of-contents)
+  - [Tested Models and Configurations](#tested-models-and-configurations)
   - [Running TGI on Gaudi](#running-tgi-on-gaudi)
   - [Running TGI with BF16 Precision](#running-tgi-with-bf16-precision)
   - [Running TGI with FP8 Precision](#running-tgi-with-fp8-precision)
+  - [TGI-Gaudi Benchmark](#tgi-gaudi-benchmark)
   - [Adjusting TGI Parameters](#adjusting-tgi-parameters)
-  - [Environment variables](#environment-variables)
+  - [Environment Variables](#environment-variables)
   - [Profiler](#profiler)
 
+
+## Tested Models and Configurations
+
+The following table contains models and configurations we have validated on Gaudi2.
+
+
+|  Model                 |  BF16        |             |  FP8         |             |
```

Comment:
Can you merge the cell for `BF16` and the empty cell after it into one? Same for `FP8` cell

Reply:
Looks like its not possible in markdown